data/en/features.yaml

```diff
@@ -13,7 +13,7 @@
   - Streaming approach enables real-time response while minimizing storage costs and complexity.
   - Ready out-of-the-box with rules, which you can customize for your environment.
 - title: Open source
-  description: A multi-vendor and broadly supported standard that you can rely on.
+  description: A multi-vendor and broadly supported solution that you can rely on.
```

Comment:
`A multi-vendor and widely adopted solution that you can rely on.`?

Reply:
added thanks!